main.go

```diff
@@ -17,21 +17,27 @@ import (
 
 const (
 	baseURL                   = "https://api.bitrise.io/v0.1"
-	buildTypeTag              = "tag"
-	buildTypePush             = "push"
-	buildTypeManual           = "manual"
-	buildTypePullRequest      = "pull-request"
 	envKeyPreviousBuildStatus = "PREVIOUS_BUILD_STATUS"
 	envKeyBuildStatusChanged  = "BUILD_STATUS_CHANGED"
 	statusTextSuccessfulBuild = "success"
 )
 
+type buildType int
+
+const (
+	buildTypeTag buildType = iota
+	buildTypePush
+	buildTypeManual
+	buildTypePullRequest
+)
+
 type config struct {
 	AppSlug             string          `env:"BITRISE_APP_SLUG,required"`
 	BuildSlug           string          `env:"BITRISE_BUILD_SLUG,required"`
 	BuildStatus         string          `env:"BITRISE_BUILD_STATUS,required"`
 	PreviousBuildStatus string          `env:"PREVIOUS_BUILD_STATUS"`
 	AccessToken         stepconf.Secret `env:"access_token,required"`
+	VerboseLog          bool            `env:"verbose_log,opt[yes,no]"`
```

Comment:
`verbose_log` for me sounds like a new variant of debug logging. please use some more commonly used one like verbose, debug, etc input ID